source/components/JsxParser.tsx

```diff
@@ -119,15 +120,21 @@ export default class JsxParser extends React.Component<TProps> {
 				this.props.onError!(new Error(`The expression '${expression.callee}' could not be resolved, resulting in an undefined return value.`))
 				return undefined
 			}
-			return parsedCallee(...expression.arguments.map(this.#parseExpression))
+			return parsedCallee(...expression.arguments.map(
+				arg => this.#parseExpression(arg, expression.callee),
+			))
 		case 'ConditionalExpression':
 			return this.#parseExpression(expression.test)
 				? this.#parseExpression(expression.consequent)
 				: this.#parseExpression(expression.alternate)
 		case 'ExpressionStatement':
 			return this.#parseExpression(expression.expression)
 		case 'Identifier':
+			if (scope?.[expression.name]) {
+				return scope?.[expression.name]
+			}
```

Comment:
That's brilliant - I love the way you've handled this case.

Reply:
😅 I realized that this would fail for falsy scope values so I had to update it. The pattern would be a nice fit for the statement below though!